javascript/chronometer.js

```diff
@@ -1,34 +1,58 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
     // ... your code goes here
+
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (callback) callback();
+    }, 1000);
   }
 
   getMinutes() {
     // ... your code goes here
+    if (this.currentTime < 60) {
+      return 0;
+    } else {
+      return Math.floor(this.currentTime / 60);
+    }
   }
 
   getSeconds() {
     // ... your code goes here
+    if (this.currentTime % 60 === 0) {
+      return 0;
+    } else {
+      return this.currentTime % 60;
+    }
   }
 
   computeTwoDigitNumber(value) {
     // ... your code goes here
+    if (value < 10) {
+      return `0${value}`;
+    } else {
+      return `${value}`;
+    }
   }
 
   stop() {
     // ... your code goes here
+    clearInterval(this.intervalId) 
   }
 
   reset() {
     // ... your code goes here
+    this.currentTime = 0
   }
 
   split() {
     // ... your code goes here
+    return `${this.computeTwoDigitNumber(this.getMinutes())}:${this.computeTwoDigitNumber(this.getSeconds().toFixed(0))}`
   }
```

Comment:
Gute Arbeit, bin gespannt wie du dich im zweiten Teil schlägst!
Hier würde ich dir empfehlen das Ergebnis in Variablen abzuspeichern, damit es etwas lesbarer/cleaner ist, also so:
```
 split() {
    let minutes = this.computeTwoDigitNumber(this.getMinutes());
    let seconds = this.computeTwoDigitNumber(this.getSeconds());

    return `${minutes}:${seconds}`;
  }
```
ist aber natürlich nicht notwendig für die Funktionalität